backend/service/k8s/lightweightinformer.go

```diff
@@ -90,7 +91,8 @@ func NewLightweightInformer(
 						if err := cacheStore.Add(lightweightObj); err != nil {
 							return err
 						}
-						h.OnAdd(d.Object)
+						// (mikecutalo) : double check what isInInitialList is doing
+						h.OnAdd(d.Object, false)
```

Comment:
i need to double check this new `isInInitialList` option but otherwise the rest is ready for review.